config/action.d/pf.conf

```diff
@@ -16,7 +16,7 @@
 # we don't enable PF automatically; to enable run pfctl -e 
 # or add `pf_enable="YES"` to /etc/rc.conf (tested on FreeBSD)
 actionstart = echo "table <<tablename>-<name>> persist counters" | pfctl -f- 
-              echo "block proto <protocol> from <<tablename>-<name>> to any port <port>" | pfctl -f-
+              echo "block proto <protocol> from <<tablename>-<name>> to any<actiontype>" | pfctl -f-
```

Comment:
Ah, I see now :-)
Please just add to the doc of actiontype that it is appended as is without any space separation (is there any negative effect from a trailing space BTW? Just wondering... Since we could also simply just strip lines on the right I guess before execution...